src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Storage {
+    private static Map<String, FruitTransaction> calculatedTransactions = new HashMap<>();
```

Comment:
```suggestion
    private static Map<String, Integer> calculatedTransactions = new HashMap<>();
```
Better parameterize the map by fruit name and quantity. Transaction object has other purpose